action.yml

```diff
@@ -1,5 +1,5 @@
-name: 'NGINX configuration sync'
-description: 'The action synchronizes NGINX configuration from a Git repository to an NGINX for Azure deployment.'
+name: 'NGINX configuration and certificate sync'
```

Comment:
I don't think we should need to update this every time we add a supported feature. Perhaps we can think of a more generic name for the action like, "NGINX For Azure Deployment Sync" or something like that?